.github/workflows/tests.yml

```diff
@@ -25,6 +25,11 @@ jobs:
           cp .env.example .env
           bin/rake test
 
+      - name: Code Climate Coverage Action
+        uses: paambaati/codeclimate-action@v3.0.0
+        env:
+          CC_TEST_REPORTER_ID: b358904aab0061aa522f50917cb78091c0fab23ab0b72bbbee3f854c00d7435c
```

Comment:
It could be but according to their docs:

> Should I keep my Test Reporter ID secret?
>
> Your repo's test coverage ID is only used to identify your repo when submitting test coverage payloads. It can't be used to access any of your data.